ingestion_tools/scripts/data_validation/conftest.py

```diff
@@ -86,7 +77,7 @@ def run_spacing_combinations(
     for run in run_names:
         for vs in voxel_spacings:
             if f"{bucket}/{dataset}/{run}/Tomograms/VoxelSpacing{vs}" in voxel_spacing_files:
```

Comment:
I believe this could be `"{bucket}/{dataset}/{run}/Tomograms/VoxelSpacing{vs:.3f}"`.  It would fix the precision at 3 for the float.

Reply:
I was thinking we could collect everything here (even the non-3f ones) and then in the voxel spacings test, we would confirm this?